internal/providers/pke/pkeworkflow/create_cluster.go

```diff
@@ -26,9 +28,24 @@ import (
 const CreateClusterWorkflowName = "pke-create-cluster"
 const pkeVersion = "0.4.8"
 
-func getDefaultImageID(region, kubernetesVersion string) string {
-	switch kubernetesVersion {
-	case "1.12.2":
+func getDefaultImageID(region, kubernetesVersion string) (string, error) {
+	constraint112, err := semver.NewConstraint(">= 1.12.0, < 1.13.0")
+	if err != nil {
+		return "", emperror.Wrap(err, "could not create semver constraint for Kubernetes version 1.12+")
+	}
+
+	constraint113, err := semver.NewConstraint(">= 1.13.0, < 1.14.0")
+	if err != nil {
+		return "", emperror.Wrap(err, "could not create semver constraint for Kubernetes version 1.13+")
+	}
+
+	kubeVersion, err := semver.NewVersion(kubernetesVersion)
+
+	if err != nil {
+		return "", emperror.WrapWith(err, "could not create semver from Kubernetes version", "kubernetesVersion", kubernetesVersion)
+	}
+
+	if constraint112.Check(kubeVersion) {
```

Comment:
I'd still use a `switch` instead of the `if`s, like
```go
switch {
	case constraint112.Check(kubeVersion):
		// ...
	case constraint113.Check(kubeVersion):
		// ...
	default:
		// ...
}
```